### ISSUE TITLE
Description property not working for TV's

### ISSUE THREAD

**Issue by hugopeek** [theboxer/Git-Package-Management#55]
```
"tvs": [
    {
        "description": "This is not being written to MODX...",
    }
]
```
If I edit the TV and enter a description manually, it is being removed when updating the package. Error log is clear.

**Comment by TheBoxer** [comment]
I can't reproduce your problem. Description for TV is same as the one in config file and it's updated when I change it in config...

**Comment by hugopeek** [comment]
Hmm, strange.. It's all on an install built with Gitify. Will try it on a fresh install too to see what happens.. Is there a debug mode or a log file somewhere, of the package update process?

**Comment by TheBoxer** [comment]
unfortunately not :)

**Comment by theboxer** [comment]
Fixed in 1159307a6667c28f43689267a654bbc9e6342a1f
